apluslms_roman/utils/path_mapping.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import PurePosixPath
+import json
+
+
+def get_host_path(original, mapping):
+    ret = original
+    orig_path = PurePosixPath(original)
+    for k, v in mapping.items():
+        try:
+            print(k, v)
+            relative_path = orig_path.relative_to(k)
+            ret = PurePosixPath(v).joinpath(relative_path)
+            return str(ret)
+        except ValueError:
+            print("Error when composing new path!")
```

Comment:
There shouldn't be `print` statements in any other file than `cli.py`. Logging can be used, when the user doesn't need to see the messages, but the messages could help him in some cases (e.g. to debug).